checks.d/yarn.py

```diff
@@ -196,14 +197,19 @@ def _yarn_app_metrics(self, rm_address, addl_tags):
 
                     for app_json in metrics_json['apps']['app']:
 
-                        app_name = app_json['name']
+                        tags = []
 
-                        tags = ['app_name:%s' % str(app_name)]
-                        tags.extend(addl_tags)
+                        for tag in app_tags:
+                            key, value = tag.split(':')
+                            tags.append("{tag}:{value}".format(
+                                tag=value, value=str(app_json[key])
```

Comment:
If `app_json[key]` doesn't exist, this will raise a `KeyError`, failing the check.

I think it would be better to either test the existence of the value before appending it or try/catch it. (and emit a log.warning without failing if there is an issue)